tests/ui/traits/const-traits/staged-api-user-crate.stderr

```diff
@@ -1,28 +1,8 @@
-error[E0015]: cannot call non-const fn `<staged_api::Unstable as staged_api::MyTrait>::func` in constant functions
+error: cannot call conditionally-const or trait fn `<staged_api::Unstable as staged_api::MyTrait>::func` in constant functions
```

Comment:
The `or` here is a bit weird. Could you just pass in the `tcx.def_descr` here so it's `conditionally-const {descr}`.

And is there a test exercising what happens when we cal a free function w/ `~const`?

Reply:
It could be a non-conditonally-const trait fn, though... but maybe that's odd enough that we can ignore it?

Reply:
Those don't exist, yes

Reply:
Ah okay, I will rename some types then.

Reply:
> And is there a test exercising what happens when we cal a free function w/ ~const?

Now there is. :)